src/main/resources/inspectionDescriptions/ClassConstantUsageCorrectnessInspection.html

```diff
@@ -1,6 +1,6 @@
 <html>
 <body>
 Reports issues related to PSR-11 strict class loading, caused by `::class` processing.
-More details <a href="https://gist.github.com/bcremer/9e8d6903ae38a25784fb1985967c6056">here</a>
+More details <a href="https://github.com/emirb/phpinspectionsea/blob/master/docs/probable-bugs.md#class-autoloading-correctness-1">here</a>
```

Comment:
Thank you for the PR Emir. Can you correct the URL please: https://github.com/kalessil/phpinspectionsea/blob/master/docs/probable-bugs.md#class-autoloading-correctness ? Currently it's leading to the fork and auto-scroll is not working.